README.md

```diff
@@ -217,10 +217,14 @@ Next, create an Azure Service Principal, which is a special type of account that
 1. Create a Service Principal with a "contributor" role that is allowed to make changes to any resources in that subscription. Replace $AZURE_SUBSCRIPTION_ID with the ID you found in step 1 and run this command:
 
 ```
-az ad sp create-for-rbac --name "CICD" --role contributor --scopes /subscriptions/$AZURE_SUBSCRIPTION_ID --sdk-auth
+az ad sp create-for-rbac  --sdk-auth --name "github-deployer" --role contributor --scopes /subscriptions/$AZURE_SUBSCRIPTION_ID
 ```
 
-1. Capture the output and add it as a [Github repository secret](/../../settings/secrets/actions/new) with the name `AZURE_CREDENTIALS`.
+1. Copy that output into your clipboard.
+1. Navigate to your repository on Github and select the _Settings_ tab.
```

Comment:
The URL that you had was a 404 for me, since I was going through the README on the microsoft account. I guess if you're positive people will read the readme in their fork, you could keep the URL, maybe there's something that forces that? Otherwise, I thought it best to give them explicit directions to find the correct place to go.

Reply:
That is a really good point. The primary use case is for this to be used _elsewhere_. Nobody should reach this section without clicking the "Use this template" button. I think we prefer to have a good experience for users that have created the template (or forked the repo) with a working link rather than making it easier for others that thought they can change settings in a separate repo. 

I don't think there is anything that we can do here to support both use cases with a nice experience :/

Reply:
Okay, so I just went through the flow again, and it does seem that the fork opens in the same tab as the original MS repo, thus students are more likely to be reading on their own fork. Should I revert the two changes to links then?

(Another approach would be sticking a note about "Make sure you're clicking that link from your fork of the template." or putting a details/summary with instructions).

Reply:
Yes, if you could revert to keep the links that would be great. Additionally, I think that adding a small note makes it even better.